validation/edepsim_validation.py

```diff
@@ -90,15 +90,158 @@ def main(sim_file, input_type):
         output.savefig()
         plt.close()
 
+        NDHallwidths = [10000.,5500.,20000.] # mm
+
+        def segment_bounds(i):
```

Comment:
This is a bit of a silly request, because it's just to do with semantics, but could you change "segment" everywhere in this function to "tpc". A "segment" has another meaning in edep-sim that could cause some confusion here.

Reply:
Sure, done